mailbox/event/event-rabbitmq/src/test/java/org/apache/james/mailbox/events/RabbitMQEventBusTest.java

```diff
@@ -791,6 +791,26 @@ void dispatchShouldPersistEventsWhenDispatchingGroupsGetErrorMultipleTimes() {
             assertThat(dispatchingFailureEvents()).containsExactly(EVENT, EVENT_2);
         }
 
+        @Test
+        void reDeliverShouldDeliverToAllGroupsWhenDispatchingFailure() {
+            EventCollector eventCollector = eventCollector();
+            eventBus().register(eventCollector, GROUP_A);
+
+            EventCollector eventCollector2 = eventCollector();
+            eventBus().register(eventCollector2, GROUP_B);
+
+            rabbitMQExtension.getRabbitMQ().pause();
+            doQuietly(() -> eventBus().dispatch(EVENT, NO_KEYS).block());
+            rabbitMQExtension.getRabbitMQ().unpause();
+            dispatchingFailureEvents()
+                .forEach(event -> eventBus().reDeliver(DispatchingFailureGroup.INSTANCE, event).block());
+
+            getSpeedProfile().shortWaitCondition()
+                .untilAsserted(() -> assertThat(eventCollector.getEvents())
+                    .hasSameElementsAs(eventCollector2.getEvents())
+                    .containsExactly(EVENT));
+        }
+
```

Comment:
Please add a test demonstrating that when redeliver to `DispatchingFailureGroup.INSTANCE` fails then the event is well stored in deadLetter again under group `DispatchingFailureGroup.INSTANCE`

Reply:
Noticing, there will be 2 events end up  at the `DispatchingFailureGroup`. One is stored for the first time dispatching, one is failed for the reDelivery. 

What I don't really like is that we are not deleting an event after successfully `reDeliver` it but instead, the responsibility is belonging to webadmin services.